### PR TITLE
Release Notes for v1.1.0

### PR DESCRIPTION
**Pull request by jparisu**
Signed-off-by: jparisu <javierparis@eprosima.com>

### CODE REVIEW

File: docs/rst/notes/notes.rst [hunk 1]
```diff
@@ -1,14 +1,34 @@
...
 ##############
-Version v1.0.0
+Version v1.1.0
 ##############
 
-This is the first stable release of *eProsima Fast DDS Monitor*.
+This release includes the following new **features**:
+
+* Add Hide/Show metatraffic entities capabilities.
+* Support cumulative data Dynamic Chart.
```
Comment (rsanchez15) [inline docs/rst/notes/notes.rst:10]:
```suggestion
* Support cumulative data in Dynamic Charts.
```

File: docs/rst/notes/notes.rst [hunk 2]
```diff
@@ -1,14 +1,34 @@
...
+
+    * Fast DDS Suite documentation.
+    * Tutorial for ROS 2 galactic.
+
+This release includes the following **bugfixes**:
+
+* Fix Y axis rendering when all values were 0.
+* Fix timestamps in Historical charts.
+* Fix deselect entity minor bugs.
+* Fix width QML warning.
```
Comment (rsanchez15) [inline docs/rst/notes/notes.rst:26]: This is not done yet but I'm on it. 

```suggestion
* Fix width QML warning.
* Fix mu(micro) symbol display in Windows
```

**Review by rsanchez15 — CHANGES_REQUESTED** [review]

File: docs/rst/notes/notes.rst (hunk 1 above)
Comment (jparisu) [reply docs/rst/notes/notes.rst:10]: :+1:

File: docs/rst/notes/notes.rst (hunk 2 above)
Comment (jparisu) [reply docs/rst/notes/notes.rst:26]: :+1:

File: docs/rst/notes/notes.rst [hunk 3]
```diff
@@ -1,14 +1,34 @@
...
+
+This release includes the following **improvements**:
+
+* Application background
+* Move last clicked logic from QML to C++.
+* Documentation improvement:
+
+    * Fast DDS Suite documentation.
+    * Tutorial for ROS 2 galactic.
+
```
[the remark this answers is not in the document]
Comment (jparisu) [reply docs/rst/notes/notes.rst:21]: :+1:

File: docs/rst/notes/notes.rst [hunk 4]
```diff
@@ -1,14 +1,36 @@
...
+This release includes the following new **features**:
+
+* Add Hide/Show metatraffic entities capabilities.
+* Support cumulative data in Dynamic Charts.
+
+This release includes the following **improvements**:
+
+* Application background
+* Move last clicked logic from QML to C++.
+* Blank text when open change alias dialog.
```
Comment (rsanchez15) [inline docs/rst/notes/notes.rst:16]:
```suggestion
* Blank text when open "*change alias*" dialog.
```

**Review by rsanchez15 — CHANGES_REQUESTED** [review]

**Review by rsanchez15 — APPROVED** [review]

**Pull request merged**